CHANGELOG.md

```diff
@@ -1,5 +1,110 @@
+* Mark `conda.gateways.disk.test.is_conda_environment` as pending deprecation. Use `PrefixData.is_environment()` instead. It will be permanently removed in 26.3. (#14750)
+* Mark `conda.gateways.disk.test.touch_nonadmin` as pending deprecation. Use `PrefixData.set_nonadmin()` instead. It will be permanently removed in 26.3. (#14750)
+* Mark `conda.base.context.Context.pip_interop_enabled` as pending deprecation, to be removed in 26.3. Use `conda.base.context.Context.prefix_data_interoperability`. instead. (#14750)
+* Mark `conda.core.prefix_data.get_python_version_for_prefix()` as pending deprecation, to be removed in 26.3. Use `conda.core.prefix_data.PrefixData(prefix).get("python").version` instead. (#14750)
+* Mark `conda.core.prefix_data.PrefixData._load_site_packages()` as pending deprecation, to be removed in 26.3. Use `conda.plugins.prefix_data_loaders.pypi.load_site_packages()` instead. (#14750)
+* Mark `conda.core.prefix_data.PrefixData._python_pkg_record` as pending deprecation, to be removed in 26.3. Use `PrefixData.get("python")` instead. (#14750)
+* Mark `conda.core.prefix_data.PrefixData` keyword argument `pip_interop_enabled` as pending deprecation, to be removed in 26.3. Use the `interoperability` keyword argument instead. (#14750)
+* Mark `conda.core.prefix_data.python_record_for_prefix()` as pending deprecation. Use `conda.core.prefix_data.PrefixData(prefix).get("python")` instead. (#14750)
+* Mark `conda.models.records.PackageRecord.schannel` as pending deprecation. Use `conda.models.records.PackageRecord.channel_name` instead. It will be permanently removed in 26.3. (#14781)
+* Mark `conda.common.pkg_formats` subpackage as pending deprecation, to be removed in 26.3. Use the `prefix_data_loaders` plugin hook instead. (#14798)
+* Mark `conda.common.pkg_formats.python` module as pending deprecation, to be removed in 26.3. Use `conda.plugins.prefix_data_loaders.pypi.pkg_format` instead. (#14798)
+* Mark `conda.gateways.disk.read.read_python_record` as pending deprecation, to be removed in 26.3. Use `conda.plugins.prefix_data_loaders.pypi.pkg_format.read_python_record` instead. (#14798)
+* Mark `conda info --license` as pending deprecation to be removed in 26.3. (#14831)
+* Mark `conda info --root` as pending deprecation to be removed in 26.3. Use `conda info --base` instead. (#14831)
+* Mark `conda.cli.main_info.get_info_components` as pending deprecation to be removed in 26.3. Use `conda.cli.main_info.iter_info_components` instead. (#14837)
+* Mark `conda.common.url.hex_octal_to_int` as pending deprecation. Use `int(..., 16)` instead. It will be permanently removed in 26.3. (#14750)
+* Mark `conda.trust.signature_verification` and `conda.trust.constants` modules for deprecation in version 25.9.0, to be removed in version 26.3.0. The functionality will be moved to `conda-content-trust` and `conda-anaconda-trust-root` packages respectively. (#14849)
```

Comment:
I think it would be advisable to sort these alphabetically so they are easier to find.